AssetRipper.TextureDecoder/Rgb/RgbConverter.cs

```diff
@@ -486,6 +486,38 @@ public static int RGB9e5FloatToBGRA32(ReadOnlySpan<byte> input, int width, int h
 			return io;
 		}
 
+		[MethodImpl(OptimizationConstants.AggressiveInliningAndOptimization)]
+		public static int R11G11B10FloatToBGRA32(ReadOnlySpan<byte> input, int width, int height, out byte[] output)
```

Comment:
I'd like a unit test for this ensuring that the struct exhibits the same behavior.
```cs
//Load one of the other RGB images.
//Convert that data to RGB32Half using the generic conversion method.
//Use this method to convert the compressed data to BGRA32.
//Use the generic conversion method to convert the compressed data to BGRA32.
//Assert equal.
```